src/components/view/DocumentDetailView/DocumentDetailView.tsx

```diff
@@ -0,0 +1,148 @@
+    const [platform, setPlatform] = useState<string>();
+    const [document, setDocument] = useState<LibraryDocument>();
+    const [loadingPreview, setLoadingPreview] = useState<boolean>(false);
+
+    const documentRef = useRef<LibraryDocument>();
+
+    const applyPreviewState = (previewState: 'loading' | DocumentDetailViewPreviewState): void => {
+        const previewPlatform = previewState.endsWith('mobile') ? 'iOS' : 'Web';
+        if (platform !== previewPlatform) {
+            setPlatform(previewPlatform);
+            return;
+        }
+
+        const previewDocument = previewState === 'loading' ? undefined : createPreviewDocument(previewState);
+        setDocument(previewDocument);
+        setLoadingPreview(previewState !== 'loading' && document?.fileUrl !== previewDocument?.fileUrl);
+    };
+
+    const loadPlatform = async (): Promise<void> => {
+        const deviceInfo = await MyDataHelps.getDeviceInfo();
+        setPlatform(deviceInfo.platform);
+    };
+
+    useInitializeView(() => {
+        if (props.previewState) {
+            applyPreviewState(props.previewState);
+            return;
+        }
+
+        if (!platform) {
+            loadPlatform();
+            return;
+        }
+
+        createSingleLibraryDocumentLoader().load(props.surveyResultId, props.surveySpecification).then(document => {
+            if (!document) {
+                MyDataHelps.back();
+            } else if (!documentRef.current) {
+                setDocument(document);
+                setLoadingPreview(!!document.fileUrl);
+            }
+        });
+    }, [], [props.previewState, platform]);
+
+    useEffect(() => {
+        documentRef.current = document;
+    }, [document]);
+
+    const onDelete = async (): Promise<void> => {
+        if (props.previewState || !document) return;
+
+        const confirmed = window.confirm(language('document-detail-view-delete-confirmation'));
+        if (confirmed) {
+            if (document.fileKey) {
+                await MyDataHelps.deleteFile(document.fileKey);
+            }
+            await MyDataHelps.deleteSurveyResult(document.surveyResultId);
+            MyDataHelps.back();
+        }
+    };
+
+    const onShare = async (): Promise<void> => {
+        if (props.previewState || !document?.fileUrl) return;
+        MyDataHelps.openEmbeddedUrl(document.fileUrl);
+    };
+
+    return <Layout colorScheme={props.colorScheme ?? 'auto'}>
+        <NavigationBar
+            showBackButton={true}
+            className="mdhui-document-detail-view-nav-bar"
+            navigationBarRight={platform && document ?
+                <div className="mdhui-document-detail-view-nav-bar-icon" onClick={onShare}>
+                    <FontAwesomeSvgIcon icon={platform === 'Web' ? faDownload : faArrowUpRightFromSquare} />
+                </div> : undefined
+            }
+        />
+        <div className="mdhui-document-detail-view">
+            {!document && <LoadingIndicator />}
```

Comment:
It looks like this isn't in a container that respects light/dark mode:
<img width="682" height="335" alt="image" src="https://github.com/user-attachments/assets/75536650-4be6-4083-ac63-eaeb9e10f228" />

Reply:
Oh - weird, it only does this on the "Docs" view.  If you use "Default" this works properly.

There's a similar quirk with the DocumentLibraryView stories:

<img width="904" height="1073" alt="image" src="https://github.com/user-attachments/assets/946f9522-6d45-40a1-8518-69c47fe8c85f" />

Reply:
Took me a little bit to figure this one out.  🙂 

These components set the `body` background color to get edge to edge color coverage.  The issue was that the docs page of the storybook inserts some elements between the `body` and these components that hard code their backgrounds to white.  Therefore, the body background was being blocked from view.

I've adjusted the CSS on these two views to set both the body background color and their own background color, so now it should render correctly in both places.